src/components/segmented-control/SegmentedControl.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { ChangeEvent, InputHTMLAttributes, useId, useState } from 'react';
+import styled from '@emotion/styled';
+import { LayoutGroup, m } from 'framer-motion';
+
+interface Props extends InputHTMLAttributes<HTMLInputElement> {
+  options: string[];
+  onChange?: (e: ChangeEvent) => void;
+}
```

Comment:
```suggestion
interface Props extends InputHTMLAttributes<HTMLInputElement> {
  options: string[];
}
```

`InputHTMLAttributes<HTMLInputElement>`에 `onChange`가 포함되어 있을 거 같아요 !